src/wallet/coinselection.h

```diff
@@ -30,9 +30,9 @@ class CInputCoin {
 
     COutPoint outpoint;
     CTxOut txout;
-    CAmount effective_value;
-    CAmount fee = 0;
-    CAmount long_term_fee = 0;
+    mutable CAmount effective_value;
+    mutable CAmount fee = 0;
+    mutable CAmount long_term_fee = 0;
```

Comment:
nit: Better to squash this change into the commit that calls for it

Reply:
Done